Makefile

```diff
@@ -82,6 +82,11 @@ docs: .venv ## Build Daft documentation
 docs-serve: .venv ## Build Daft documentation in development server
 	JUPYTER_PLATFORM_DIRS=1 uv run mkdocs serve -f mkdocs.yml
 
+.PHONY: frontend
+frontend: .venv ## Build daft-dashboard frontend assets
+	cd src/daft-dashboard/frontend && \
+		bun run build
```

Comment:
👏  thank you!

Is this not a pre-requisite to other tasks? My builds fail from a clean env, perhaps this is a build dep.

Reply:
It doesn't look `bun run build` checks if the previous assets are up to date or not, so if I made it a pre-req to the build itself it would add another 8 seconds to it. If that's OK, happy to add it. The error doesn't appear for `make build` but it does for the others like `make build-release`

Reply:
Let's add it as a build-release pre-requisite

Reply:
Added